# Message containing all the info for approaching the user

bool status
bool move
float32 angle